backend/app/assets/javascripts/spree/backend/admin.js.erb

```diff
@@ -7,6 +7,7 @@
 //= require spree/backend/product_picker
 //= require spree/backend/option_value_picker
 //= require spree/backend/taxons
+//= require spree/backend/country_states
```

Comment:
True, but the frontend javascript was not suitable for the backend so I had to clone and edit this for the backend.
However there are now 2 country/states javascripts in the backend. Which the one I created has some more functionality, so maybe we should deprecate the old one.